website_docs/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Ruby"
+weight: 24
+description: >
+  <img width="35" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Ruby_SDK.svg"></img>
+  A language-specific implementation of OpenTelemetry in Ruby.
+---
+
+This is the OpenTelemetry for Ruby documentation. OpenTelemetry is an
+observability framework -- an API, SDK, and tools that are designed to aid in
+the generation and collection of application telemetry data such as metrics,
+logs, and traces. This documentation is designed to help you understand how to
+get started using OpenTelemetry for Ruby.
+
+## Status and Releases
+
+The current status of the major functional components for OpenTelemetry Ruby is
+as follows:
+
+| Tracing | Metrics | Logging |
+| ------- | ------- | ------- |
+| Beta    | Alpha   | Not Yet Implemented |
```

Comment:
My only nit is we probably are better off saying `Metrics` is not yet implemented, as the api is so out of date as to be unusable, and i think we plan to start from close to scratch. @mwear @fbogsany do i have that correct?

otherwise lgtm and i think this is great for discoverability

Reply:
yep - we removed the Metrics API in #697 and we'll start from scratch

Reply:
```suggestion
| Beta    | Not Yet Implemented | Not Yet Implemented |
```